src/pycram/datastructures/enums.py

```diff
@@ -205,6 +205,16 @@ class MJCFJointType(Enum):
     FIXED = "fixed"  # Added for compatibility with PyCRAM, but not a real joint type in MuJoCo.
 
 
+class MovementType(Enum):
+    """
+    Enum for the different movement types of the robot.
+    """
+    STRAIGHT_TRANSLATION = auto()
```

Comment:
From the context it is not understandable to me what these enums fields mean. Please provide docsstring for them